tests/robustness/client.go

```diff
@@ -36,8 +36,8 @@ func NewClient(endpoints []string, ids identity.Provider, baseTime time.Time) (*
 	cc, err := clientv3.New(clientv3.Config{
 		Endpoints:            endpoints,
 		Logger:               zap.NewNop(),
-		DialKeepAliveTime:    1 * time.Millisecond,
-		DialKeepAliveTimeout: 5 * time.Millisecond,
+		DialKeepAliveTime:    10 * time.Second,
```

Comment:
```
	// DialKeepAliveTime is the time after which client pings the server to see if
	// transport is alive.
```
This is pretty big value, as it's greater then 2 seconds, isn't it just saying don't do any keep alives?

Reply:
The gRPC client will update it to 10s if it is smaller than 10s. 

https://github.com/grpc/grpc-go/blob/eeb9afa1f6b6388152955eeca8926e36ca94c768/dialoptions.go#L445

Reply:
ok, so maybe we can just skip it as it's not really useful config

Reply:
The etcd client pkg only uses keepalive grpc opt when the time is large than 0. So we still need this setting.

Reply:
Oh, thanks for pointing this out.